jdbc/src/main/resources/interpreter-setting.json

```diff
@@ -115,6 +115,13 @@
         "defaultValue": false,
         "description": "Enable ZeppelinContext variable interpolation into paragraph text",
         "type": "checkbox"
+      },
+      "zeppelin.jdbc.maxConnLifetime": {
+        "envName": null,
+        "propertyName": "zeppelin.jdbc.maxConnLifetime",
+        "defaultValue": -1,
```

Comment:
Please add double quotes near `-1`.
Otherwise it will be converted in "-1.0" and will cause NumberFormatException.

Reply:
Fixed